app/assets/javascripts/widgets/brochure_masthead.js

```diff
@@ -0,0 +1,6 @@
+$(document).ready(function() {
+  $(".offcanvas-menu-trigger a").on("click", function(){
+    $("#js-cu-off-canvas-nav-trigger").trigger("click");
+    return false;
```

Comment:
This code looks odd to me. Shouldn't this be covered by existing OmniNav functionality?